src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,32 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={selectedTabId === tab ? 'is-active' : ''}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                onTabSelected(tab);
```

Comment:
you must not invoke this func if you click on already selected tab and also check if you pass here not existing tab you must choose first one